tests/foreman/cli/test_host.py

```diff
@@ -2163,6 +2164,35 @@ def _register_client(
                 result = self.client.run('subscription-manager attach --auto')
 
         if attach_to_default:
+
+            # Remove repository-set from SetUpClass to have no repositories available
+            # Firstly the content view needs to be removed
+            default_cv = ContentView.info({'name': DEFAULT_CV, 'organization-id': self.org['id']})
+            default_lce = LifecycleEnvironment.info(
+                {'name': ENVIRONMENT, 'organization-id': self.org['id']}
+            )
+            ContentView.remove(
```

Comment:
Is it smart to remove contentview and disable repository set? Don't you modify global config this way? What about next test that uses `_register_client`, but with `attach_to_default` set to `False`? Would it still pass?

Reply:
As I mentioned in description of this PR,  there is only one test which use attach_to_default. 
And as each test creates his own [host ](https://github.com/SatelliteQE/robottelo/pull/8023/files#diff-c5074666cd0572f014a9882591a5cce5L2115). I think there should be no problem with that. 

But what I do not like is that in _register client is an option that is used only for one test, so in the future when another test will be added, another work would be required.

I will run multiple test and I will provide the result.

Reply:
@mirekdlugosz  ^^

Reply:
OK, this reassurance is all I needed.